portfolio/src/main/webapp/comments-script.js

```diff
@@ -51,3 +54,8 @@ function createRowElement() {
     const rowElement = document.createElement('tr');
     return rowElement;
 }
+
+function changeCommentsPerPage(){
+    commentsPerPage = document.getElementById("comments-per-page").value -1;
```

Comment:
nit, but space around '-' should probably be consistent on each side of the minus sign.